wpeverest/everest-forms
# Fix - conflict between error messages and move confirm field

### All Submissions:






### Changes proposed in this Pull Request:



This PR fixes the conflict between required field message and confirm field message. It also moves the confirm field validation codes from pro to here, reason being the free version field `Email` also uses a confirm field, which will not work if the pro version of Everest Forms is deactivated. But this PR solves that problem.

### How to test the changes in this Pull Request:

1. Create a new form or Edit existing form.
2. Add `Email` field and enable `Email Confirmation`.
3. Make sure to deactivate the pro version of Everest Forms.
4. Go to the frontend and see if the email confirmation message is working properly.
5. Then add a `Password` field.
6. Enable password confirmation and make it required.
7. Go to frontend and see if the password confirmation and required field messages are working as per requirements.

### Types of changes:




### Other information:




### Changelog entry

> Fix - Conflict between required field message and confirm field message. Also move the confirm field validation codes from pro to here.
